Jenkinsfile

```
@@ -51,110 +144,99 @@ def gitMerge(String primaryBranch) {
-      [ os: "windows", browser: "MicrosoftEdge" ],
-      [ os: "macos", browser: "safari" ],
-      [ os: "macos", browser: "chrome" ],
-      [ os: "macos", browser: "firefox" ]
-    ]
-
-    def cleanAndInstall = {
-      echo "Installing tools"
-      exec("git clean -fdx modules scratch js dist")
+    stage('Install') {
       yarnInstall()
     }
 
-    def processes = [:]
-
-    // Browser tests
-    for (int i = 0; i < platforms.size(); i++) {
-      def platform = platforms.get(i)
-
-      def buckets = platform.buckets ?: 1
-      for (int bucket = 1; bucket <= buckets; bucket++) {
-        def suffix = buckets == 1 ? "" : "-" + bucket
-
-        // closure variable - don't inline
-        def c_bucket = bucket
-
-        def name = "${platform.os}-${platform.browser}${suffix}"
-
-        processes[name] = {
-          stage(name) {
-            node("bedrock-${platform.os}") {
-              echo("Bedrock tests for ${name}")
-
-              echo("Checking out code on build node: $NODE_NAME")
-              checkout(scm)
-
-              // windows tends to not have username or email set
-              tinyGit.addAuthorConfig()
-              gitMerge(primaryBranch)
-
-              cleanAndInstall()
-              exec("yarn ci")
-
-              echo("Running browser tests")
-              runBrowserTests(name, platform.browser, platform.os, c_bucket, buckets, runAllTests)
-            }
-          }
-        }
-      }
-    }
-
-    processes["headless-and-archive"] = {
-      stage("headless tests") {
-        runHeadlessTests(runAllTests)
-      }
-
-      if (env.BRANCH_NAME != primaryBranch) {
-        stage("Archive Build") {
-          exec("yarn tinymce-grunt prodBuild symlink:js")
-          archiveArtifacts artifacts: 'js/**', onlyIfSuccessful: true
-        }
+    stage('Type check') {
+      withEnv(["NODE_OPTIONS=--max-old-space-size=1936"]) {
+        exec("yarn ci-all-seq")
       }
     }
 
-    stage("Install tools") {
-      cleanAndInstall()
+    stage('Moxiedoc check') {
+      exec("yarn tinymce-grunt shell:moxiedoc")
     }
+  }
 
-    stage("Type check") {
-      withEnv(["NODE_OPTIONS=--max-old-space-size=1936"]) {
-        exec("yarn ci-all-seq")
+  // Local nodes use os: windows | macos; Remote tests use os full name e.g.: macOS Sonoma
+  def platforms = [
+    // Local tests
+    // [ browser: 'edge', os: 'windows' ],
+    // [ browser: 'firefox', os: 'macos' ],
+    // Remote tests
+    [ browser: 'chrome', provider: 'aws', buckets: 2 ],
+    // [ browser: 'edge', provider: 'aws', buckets: 2 ], // TINY-10540: Investigate Edge issues in AWS
+    [ browser: 'firefox', provider: 'aws', buckets: 2 ],
+    [ browser: 'edge', provider: 'lambdatest', buckets: 1 ],
+    [ browser: 'chrome', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ],
+    [ browser: 'firefox', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ],
+    [ browser: 'safari', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ]
```

```suggestion
    // [ browser: 'edge', provider: 'lambdatest', buckets: 1 ],
    // [ browser: 'chrome', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ],
    // [ browser: 'firefox', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ],
    // [ browser: 'safari', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ]
```
Disable Mac and Microsoft Edge tests